src/item.h

```diff
@@ -221,17 +238,49 @@ class item : public JsonSerializer, public JsonDeserializer
          * items that stack together (@ref stacks_with).
          */
         bool merge_charges( const item &rhs );
- void put_in(item payload);
- void add_rain_to_container(bool acid, int charges = 1);
 
  int weight() const;
 
  int precise_unit_volume() const;
  int volume(bool unit_value=false, bool precise_value=false) const;
- int volume_contained() const;
- int attack_time() const;
- int damage_bash() const;
- int damage_cut() const;
+
+    /**
+     * @name Melee
+     *
+     * The functions here assume the item is used in melee, even if's a gun or not a weapon at
+     * all. Because the functions apply to all types of items, several of the is_* functions here
+     * may return true for the same item. This only indicates that it can be used in various ways.
+     */
+    /*@{*/
+    /**
+     * Number of moves (@ref Creature::moves) that a single attack take.
```

Comment:
Should probably be "base number of moves". Attack time is heavily dependent on attacker.